src/applications/letters/utils/helpers.jsx

```diff
@@ -536,3 +535,14 @@ export function isAddressEmpty(address) {
     true,
   );
 }
+
+export function getDownloadSuccessMessage(optionsLength) {
+  let message;
+  if (optionsLength === 1) {
+    message = `Your letter includes the one item you selected.`;
+  } else {
+    message = `Your letter includes the ${optionsLength} items you selected.`;
+  }
+
+  return message;
+}
```

Comment:
Suggesting a shorter version of this to reduce a couple lines of code:

```suggestion
export function getDownloadSuccessMessage(optionsLength) {
  return `Your letter includes the ${optionsLength === 1 ? 'one item' : `${optionsLength} items`} you selected.`;
}
```

Alternatively, if that's not as readable as we want it, we can try this instead:

```suggestion
export function getDownloadSuccessMessage(optionsLength) {
  if (optionsLength === 1) {
    return 'Your letter includes the one item you selected.';
  }
  
  return `Your letter includes the ${optionsLength} items you selected.`
}
```